03/README.md

```diff
@@ -0,0 +1,134 @@
+# Lesson 3
+
+## Content
+- Classes in Kotlin
+  - Normal class
+  - Data class
+- Abstract class
+- Interface
+
+## Useful links
+- [Slack channel](https://appport-academy.slack.com/) (invite can be requested via [event link](https://appport.cz/event/appport-academy/) or [Meetup group](https://www.meetup.com/Prague-Mobile-Development-Meetup/events/269765161/))
+- [Github](https://github.com/Concur-Kotlin-Academy/academy2020)
+- Kotlin manual: [link](https://kotlinlang.org/docs/reference/classes.html)
+
+## A little bit of theory
+### Object Oriented Programming (OOP)
+> Object oriented programming allows us to solve complex problems by using objects
+
+More info in [Wikipedia](https://en.wikipedia.org/wiki/Object-oriented_programming)
+
+
+### Classes
+#### What's a class?
+**CLASS** is a blueprint or a set of instructions to build a specific type of object.
+
+![Class](images/class.png)
+
+##### Parts of a class
+1. **Name:** Identifies the class
+2. **Constructor(s):** Function used to initialize the state/properties of newly created objects of the class. Invoked when creating an instance of the class
+3. **Attribute(s)/property(ies):** keep state
+4. **Method(s)/function(s):** Do something
+
+##### Visibility modifiers in Kotlin
+Classes, objects, interfaces, constructors, functions, properties and their setters can have visibility modifiers.
+
+- `private` - visible (can be accessed) from inside the class only.
+- `protected` - visible to the class and its subclass.
+- `internal` - any client inside the module can access them.
+- `public` - visible everywhere.
+
+> The default visibility, used if there is no explicit modifier, is **public**.
+
+
+#### What are objects?
+**OBJECT** is an instance of a class. An object is nothing but a self-contained component which consists of methods and properties to make a particular type of data useful
+
+![Objects](images/objects.png)
+
+#### What's the difference between class and object?
+- A class is a blueprint or prototype that defines the variables and the methods (functions) common to all objects of a certain kind.
+- An object is a specimen of a class. Software objects are often used to model real-world objects you find in everyday life.
+
+## Tasks
+### Project creation
+1. Open IntelliJ IDEA
+2. Select create new project
+3. Choose `Gradle -> Kotlin/JVM`
+4. Name the project `lesson-03`
+5. Create a new Kotlin file in the `src` folder with name `Main`
+6. Add the following Kotlin code
+
+    ```kotlin
+    fun main(){
+        println("Hello World")
+    }
+    ```
+7. Right click -> run `MainKt` and check that it works
+
+
+### Working with classes
+#### Task 1
+1. Convert the previous `Hello world` implementation to an Objected-oriented version
+
+#### Task 2
+1. Define the class `Car` according to the image above
```

Comment:
This task might be actually split to multiple if you would like to show how to use primary, secondary constructors, init... Each class could be different car model. E.g. FabiaCar for simplest one and SuperbCar for most complex class ;-)

Reply:
i was considering that, but i think it would be too much for this lesson :-)